### ISSUE TITLE
Python 3.6 Support

### ISSUE THREAD

**Issue by href** [phlax/scrambler#2]
I can't seem to install scrambler on Python 3.6:

```python
Collecting scrambler
  Downloading https://.../root/pypi/+f/606/22fa21a95c84b/scrambler-0.0.4.tar.gz
Collecting distribute (from scrambler)
  Downloading https://.../root/pypi/+f/c6c/59594a7b180af/distribute-0.7.3.zip (145kB)
    100% |████████████████████████████████| 147kB 5.1MB/s
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 3, in <module>
      File "/private/var/folders/66/rpvt11kd3v145gq0v8cc5zk00000gn/T/pip-build-zz4rlg8k/distribute/setuptools/__init__.py", line 2, in <module>
        from setuptools.extension import Extension, Library
      File "/private/var/folders/66/rpvt11kd3v145gq0v8cc5zk00000gn/T/pip-build-zz4rlg8k/distribute/setuptools/extension.py", line 5, in <module>
        from setuptools.dist import _get_unpatched
      File "/private/var/folders/66/rpvt11kd3v145gq0v8cc5zk00000gn/T/pip-build-zz4rlg8k/distribute/setuptools/dist.py", line 7, in <module>
        from setuptools.command.install import install
      File "/private/var/folders/66/rpvt11kd3v145gq0v8cc5zk00000gn/T/pip-build-zz4rlg8k/distribute/setuptools/command/__init__.py", line 8, in <module>
        from setuptools.command import install_scripts
      File "/private/var/folders/66/rpvt11kd3v145gq0v8cc5zk00000gn/T/pip-build-zz4rlg8k/distribute/setuptools/command/install_scripts.py", line 3, in <module>
        from pkg_resources import Distribution, PathMetadata, ensure_directory
      File "/private/var/folders/66/rpvt11kd3v145gq0v8cc5zk00000gn/T/pip-build-zz4rlg8k/distribute/pkg_resources.py", line 1518, in <module>
        register_loader_type(importlib_bootstrap.SourceFileLoader, DefaultProvider)
    AttributeError: module 'importlib._bootstrap' has no attribute 'SourceFileLoader'
```

The problem seems to be distribute, which cannot be installed. I've noticed that setuptools was replaced by distribute [a while back](https://github.com/phlax/scrambler/commit/d851d021bb192c2465dd0e9e955711664f1970d6). Do you remember the reason for this? I noticed that by switching back to setuptools the problem seems to be solved and scrambler seems to still work.

**Comment by href** [comment]
Any way we can get this fixed? I've been using my fork that switches back to setuptools without any issues: https://github.com/href/scrambler

**Comment by phlax** [comment]
i dont rem the reason, but im pretty sure switching back to setuptools is ok - can you PR @href ?

**Comment by href** [comment]
Sure

**Comment by href** [comment]
Fixed with Version 0.0.5 now available on PyPI.
